bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/handler/LGWebOSHandler.java

```diff
@@ -127,7 +130,9 @@ public void initialize() {
         s.setListener(this);
         socket = s;
 
+        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE, "TV is off");
         startReconnectJob();
+        findMacAddress(); // host config value could have changed
```

Comment:
Is it necessary to call it twice, once here and once more after web socket connection ?
Is this one really necessary ?

Reply:
Initially thought this should cover the case when config gets updated and TV  is offline. But in this case arping won't return anthing. removing it from initialize